packages/trader/src/AppV2/Components/Guide/decription-modal.tsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import { ActionSheet, Heading, Chip, Text } from '@deriv-com/quill-ui';
+import { Localize } from '@deriv/translations';
+import { observer, useStore } from '@deriv/stores';
+import { clickAndKeyEventHandler } from '@deriv/shared';
+import { AVAILABLE_CONTRACTS, CONTRACT_LIST, getDescriptionVideoId } from 'AppV2/Utils/trade-types-utils';
+import VideoPlayer from 'App/Components/Elements/VideoPlayer';
+import TradeDescription from './Description/trade-description';
+import VideoPreview from './Description/video-preview';
+
+type TDescriptionModal = {
+    is_open?: boolean;
+    onClose: () => void;
+    onChipSelect: (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void;
+    onTermClick: (term: string) => void;
+    selected_contract_type: string;
+};
+
+const DescriptionModal = observer(
+    ({ is_open, onClose, onChipSelect, onTermClick, selected_contract_type }: TDescriptionModal) => {
+        const [is_video_player_opened, setIsVideoPlayerOpened] = React.useState(false);
+        const modal_ref = React.useRef<HTMLDialogElement>(null);
+
+        const { ui } = useStore();
+        const { is_dark_mode_on } = ui;
+
+        const video_src = getDescriptionVideoId(selected_contract_type, is_dark_mode_on);
+        //TODO: temporary, until we'll have ordered list, coming from contract type selection
```

Comment:
Waiting for contract type selection in order to get ordered (sorted) list of contracts from there